samples_project/Assets/SampleViewer/Samples/measure/measure.cs

```diff
@@ -0,0 +1,322 @@
+﻿// Copyright 2022 Esri.
```

Comment:
The copy right for the Unreal samples has some extra lines, don't know if they should be the same.

/* Copyright 2022 Esri
 *
 * Licensed under the Apache License Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */